list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+$query = "SELECT * FROM books";
+$result = mysqli_query($connection, $query);
```

Comment:
**Observations:** The script directly queries the database and renders HTML, which can lead to tightly coupled code that is difficult to maintain.

**Recommendations:**
- **Refactor to Use a Model and Repository:**
  - Implement a `Book` model and a `BookRepository` class to handle data operations.
  - Use the repository to fetch data and pass it to a view for rendering.

- **Implement Pagination:**
  - Consider adding pagination to handle large datasets efficiently.

- **Enhance Security:**
  - Escape output data to prevent XSS attacks, even though PHP's `echo` statements are used here, it's a good practice.
  - Example:
    ```php
    echo htmlspecialchars($row['title']);
    ```